vector/view.go

```diff
@@ -52,19 +54,19 @@ func NewView(val Any, index []uint32) Any {
 	return &View{val, index}
 }
 
-func NullsView(nulls *Bool, index []uint32) *Bool {
-	if nulls == nil {
+func BoolView(vec *Bool, index []uint32) *Bool {
```

Comment:
Nit: Callt this NewBoolView so it starts with "New" like the other vector constructors.